app/controllers/ems_common.rb

```diff
@@ -711,6 +711,7 @@ def set_form_vars
     @edit[:new][:hostname] = @ems.hostname
     @edit[:new][:emstype] = @ems.emstype
     @edit[:amazon_regions] = get_amazon_regions if @ems.kind_of?(ManageIQ::Providers::Amazon::CloudManager)
+    @edit[:google_regions] = get_google_regions if @ems.kind_of?(ManageIQ::Providers::Google::CloudManager)
```

Comment:
Oh I got confused...I was thinking of instance types (i.e. flavors), which is not a part of their API.

We did this for regions because we have a bit of a chicken/egg problem, if I recall.  Before we even have an EMS, we need the set of regions to show in the "Add an EMS" page, so the user can choose it.  We thought it might be expensive to go live to get the region list.  Additionally, certain regions are not in the payload (like GovCloud), so we'd have to manipulate it anyway.

I also tried with the newer v2 version of the gem, and I can't even find *how* to query for it :(